cmd/virt-manifest/virt-manifest.go

```diff
@@ -0,0 +1,55 @@
+package main
+
+import (
+	"flag"
+	"fmt"
+	"net/http"
+	"strconv"
+	"time"
+
+	"github.com/emicklei/go-restful"
+	"github.com/libvirt/libvirt-go"
+
+	"kubevirt.io/kubevirt/pkg/logging"
+	"kubevirt.io/kubevirt/pkg/virt-handler/virtwrap"
+	"kubevirt.io/kubevirt/pkg/virt-manifest/rest"
+)
+
+func main() {
+	logging.InitializeLogging("virt-manifest")
+	libvirt.EventRegisterDefaultImpl()
+	libvirtUri := flag.String("libvirt-uri", "qemu:///system", "Libvirt connection string.")
+	libvirtUser := flag.String("user", "", "Libvirt user")
+	libvirtPasswd := flag.String("pass", "", "Libvirt password")
+	listen := flag.String("listen", "0.0.0.0", "Address where to listen on")
+	port := flag.Int("port", 8186, "Port to listen on")
+	flag.Parse()
+
+	log := logging.DefaultLogger()
+	log.Info().Msg("Starting virt-manifest server")
+
+	log.Info().Msg("Connecting to libvirt")
+	// FIXME: this connection should be closed immediately
```

Comment:
It turns out that input checking was enough to prevent the bad XML that caused our code to close the libvirt connection. Since we can now re-use the connection, this issue is moot.